build/PackageDiffIgnore.xml

```diff
@@ -9384,7 +9384,11 @@
 			<Member fullName="System.Void Windows.Devices.Geolocation.Geolocator.OnProviderDisabled(System.String provider)" reason="Not present in Windows" />
 			<Member fullName="System.Void Windows.Devices.Geolocation.Geolocator.OnProviderEnabled(System.String provider)" reason="Not present in Windows" />
 			<Member fullName="System.Void Windows.Devices.Geolocation.Geolocator.OnStatusChanged(System.String provider, Android.Locations.Availability status, Android.OS.Bundle extras)" reason="Not present in Windows" />
-			<Member fullName="System.Void Windows.Devices.Lights.Lamp..ctor()" reason="Not present in Windows" />
+			<Member fullName="System.Void Windows.UI.Xaml.Controls.TextBox.OnInputScopeChanged(Windows.UI.Xaml.DependencyPropertyChangedEventArgs e)" reason="Changed parameter type and made private." />
+			<Member fullName="System.Void Windows.UI.Xaml.Controls.TextBox.OnIsSpellCheckEnabledChanged(Windows.UI.Xaml.DependencyPropertyChangedEventArgs e)" reason="Changed parameter type and made private." />
+			<Member fullName="System.Void Windows.UI.Xaml.Controls.TextBox.OnIsTextPredictionEnabledChanged(Windows.UI.Xaml.DependencyPropertyChangedEventArgs e)" reason="Changed parameter type and made private." />
+			<Member fullName="System.Void Windows.UI.Xaml.Controls.TextBox.OnTextAlignmentChanged(Windows.UI.Xaml.DependencyPropertyChangedEventArgs e)" reason="Changed parameter type and made private." />
+      <Member fullName="System.Void Windows.Devices.Lights.Lamp..ctor()" reason="Not present in Windows" />
```

Comment:
This looks incorrect or a merge duplicate. I think this is coming from your PR @MartinZikmund ?

Reply:
@jeromelaban It is coming from my PR, but I think you are looking only at the merge commit.

Reply:
Yes, that's correct, that is only the merge commit